<launch>

  <!-- start the data stream -->
  <include file="$(find aravis_camera_driver)/launch/basler_ace.launch"/> 

  <!-- Launch linear rail node which provides services-->
  <node pkg="circle" type="linearRail" name="LinearRail" output="screen">
	<param name="image_base_name" type="string" value="P0" />
	<param name="image_dir_name" type="string" value="/home/clewis/images" />
        <remap from="CameraImage" to="/image_color" />
  </node>


</launch>